recipe/patches/0001-linux64-update-flags-zstd.patch

```diff
@@ -0,0 +1,19 @@
+--- src/runtime/Config.x86-64-linux     2024-04-29 16:18:52.000000000 -0500
++++ src/runtime/Config.x86-64-linux     2024-05-27 18:47:57.623786724 -0500
+@@ -37,14 +37,14 @@
+ endif
+
+ ifdef LISP_FEATURE_SB_CORE_COMPRESSION
+-  OS_LIBS += -lzstd
++  OS_LIBS += -L${PREFIX}/lib -lzstd
+ endif
+
+ ifdef HAVE_LIBUNWIND
+   OS_LIBS += -lunwind
+ endif
+
+-CFLAGS += -Wunused-parameter -fno-omit-frame-pointer -momit-leaf-frame-pointer
++CFLAGS += -Wunused-parameter -fno-omit-frame-pointer -momit-leaf-frame-pointer
```

Comment:
This does nothing, right? In that case, the zstd-flags patches can be combined between linux-64 and linux-aarch64?

Reply:
Correct, it's an oversight from the debugging. Yes, we could combine a few patches, though it may render the selector a bit long. I combined the x86_64 and aarch64 zstd, but kept the osx and win separate. The cross selector should not be applied when cross-compiling on windows
I removed the `ppc64le`, I have no clue as to why it does not work after working on it for quite a bit, I am hoping something will come from the SBCL community, so likely it'll be a while before it can be integrated